dataflux_core/download.py

```diff
@@ -78,7 +78,7 @@ def compose(
     if storage_client is None:
         storage_client = storage.Client(
             project=project_name,
-            client_info=ClientInfo(user_agent="dataflux/0.0"),
+            client_info=ClientInfo(user_agent=user_agent.user_agent_string),
```

Comment:
This should work, but an equivalent approach should actually be to get rid of this line entirely, then move "user_agent.add_dataflux_user_agent(storage_client)" below outside of the else block so that it always runs